### PR TITLE
Vector Dialect Examples

### PR DESCRIPTION
**Pull request by Ris-Bali**
Examples for operations in vector dialect .

### CODE REVIEW

**Comment by Ris-Bali** [comment]
Done ! Pls Review

**Comment by meshtag** [comment]
Please ping me one you are done adding all examples you want to, I'll start the review process then.

**Comment by Ris-Bali** [comment]
@meshtag I'm done with all the examples please review

**Comment by meshtag** [comment]
I had a look at the code, a generic suggestion from my side for making variable naming more obvious is : 
 - Use %cons1 to represent a constant with value 1 (not any other value)
 - Please take care of minor formatting issues as well (space after comma, newline at the end of each file etc.)
 
 I'll try to provide a more detailed review and test the code once I get some time.

**Comment by Ris-Bali** [comment]
I think I have addressed the formatting errors pls review

File: examples/MLIRVector/vector-maskedstore.mlir [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+func.func private @printMemrefI32(memref<*xi32>)
+
+func.func @main() {
+  %mask0 = arith.constant dense<[0, 1, 0, 1, 0, 1, 0, 1, 1]> : vector<9xi1>
+  %value0 = arith.constant dense<[12, 78, 54, 32, 34, 36, 39, 90, 21]> : vector<9xi32>
+
+  %base = memref.get_global @gv : memref<3x3x3xi32> 
+  %c0 = arith.constant 0 : index
+
+  %c1 = arith.constant 2 : index
```
Comment (meshtag) [inline examples/MLIRVector/vector-maskedstore.mlir:13]: This does not follows the name convention, please rectify it.

**Review by meshtag — CHANGES_REQUESTED** [review]
I have added a commit addressing whitespace issues. Please go through it carefully and check if any other similar issues are to be addressed.

**Comment by Ris-Bali** [comment]
@meshtag I have addressed all your formatting isuues please review

**Pull request merged**